lang/cs/Org.Apache.REEF.Driver/Task/TaskStates.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+// 
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+namespace Org.Apache.REEF.Driver.Task
+{
+    /// <summary>
+    /// Task states in task transitions
+    /// </summary>
+    public enum TaskStates
```

Comment:
Maybe call this `TaskState` and `TaskEvent`? It makes for an easier read where it is used. Right now, those cases read as if they are collections because of the plural form.